lib/pkgcloud/openstack/database/client/instances.js

```diff
@@ -320,3 +320,70 @@ exports.setVolumeSize = function setVolumeSize(instance, newSize, callback) {
     }), callback);
   });
 };
+
+// Enable the root user for the given database instance.
+// ### @instance {string | Object} The ID of the instance or a instance or Instance class (required)
+// ### @callback {Function} Function to continue, the call is cb(error, res)
+exports.enableRootUser = function(instance, callback) {
+  // Check for instance
+  if (typeof instance === 'function' || typeof instance === 'undefined') {
+    return errs.handle(errs.create({
+      message: 'An instance is required.'
+    }), Array.prototype.slice.call(arguments).pop());
+  }
+
+  var instanceId = instance instanceof Instance ? instance.id : instance;
+
+  var options = {
+    method: 'POST',
+    path: 'instances/' + instanceId + '/root'
+  };
+
+  this._request(options, function (err, body, response) {
+    if (err) {
+      return callback(err);
+    }
+
+    if (response.statusCode === 200) {
+      return callback(null, body);
```

Comment:
I think we're probably over-engineering at this point. Returning body alone should be fine, since a change to the response will probably break a user, but will certainly break pkgcloud.